src/ForgotPasswordController.js

```diff
@@ -74,17 +78,25 @@ function (Okta, FormController, Enums, FormType, ValidationUtil, ContactSupport,
           });
         })
         .fail(function () {
-          self.set('factorType', Enums.RECOVERY_FACTOR_TYPE_EMAIL);
```

Comment:
should we remove the whole ```.fail``` code? It seems we don't need it anymore

Reply:
👍  will do

Reply:
Actually, we need to leave this in. It breaks some tests when i take it out.

Reply:
Then maybe we need to handle the error and show an error message?

Reply:
It never showed an error message before.. I think it is handled at another level

Reply:
Ok. Can you then just add a comment on why we don't need to process the error? That will help in the future to know why that empty fail is there (same comment for the the unlock controller)